src/main/resources/static/views/investment/ProjectInvestments.html

```diff
@@ -0,0 +1,31 @@
+<div ng-controller="ProjectController"
+     style="max-height: 210px; overflow: auto;">
+	<ul ng-repeat="investment in project.dtoInvestments"
+	    class="list-group checked-list-box">
+		<!--ng-if="project.investor.email == currentUser."-->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)